extensions/amp-form/0.1/amp-form.js

```diff
@@ -1007,12 +1013,13 @@ export class AmpForm {
    * @private
    */
   assertSsrTemplate_(value, msg) {
-    const supported = this.ssrTemplateHelper_.isSupported();
-    userAssert(
-      supported === value,
-      '[amp-form]: viewerRenderTemplate | %s',
-      msg
-    );
+    this.ssrTemplateHelper_.isSupported().then(supported => {
+      userAssert(
```

Comment:
This has changed from a sync assert (a throw) to a possibly rejected promise. You need to wait on the value.

Reply:
Yeah, I just spotted this as well. When I wrote this, I didn't know that `userAssert` throws. Working on it now 👍

Reply:
meh, my solution stinks. I'll take another look tomorrow to see if I can make it cleaner.